.circleci/config.yml

```
@@ -0,0 +1,38 @@
+references:
+  install_bundler: &install_bundler
+    run:
+      name: Install bundler
+      command: gem install bundler:1.14.5
+  wait_for_postgres: &wait_for_postgres
+    run:
+      name: Wait for Postgres
+      command: dockerize -wait tcp://localhost:5432 -timeout 1m
+  setup: &setup
+    run:
+      name: Set up the environment
+      command: bin/setup
+  run_tests: &run_tests
+    run:
+      name: Run the test suite
+      command: COVERAGE=true bin/rake
+
+jobs:
+  build:
+    working_directory: ~/radfords
+    docker:
+      - image: "cimg/ruby:2.5-browsers"
+        environment:
+          PGHOST: localhost
+          PGUSER: radfords
+          RAILS_ENV: test
+      - image: "cimg/postgres:14.4"
+        environmet:
```

```suggestion
        environment:
```

If it's this I will be mad!